angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts

```diff
@@ -103,10 +103,6 @@ export const defaultStory = () => ({
               text: 'Opt Out',
               href: '#nogo'
             },
-            {
-              text: 'About That',
```

Comment:
This wasnt in the examples for the other storybooks so while i was in here i removed it so they all look the same content wise in storybook